app/grandchallenge/reader_studies/serializers.py

```diff
@@ -79,6 +79,16 @@ class DisplaySetSerializer(HyperlinkedModelSerializer):
     view_content = JSONField(
         source="reader_study.view_content", read_only=True
     )
+    index = SerializerMethodField()
+
+    def get_index(self, obj):
+        if obj.reader_study.shuffle_hanging_list:
+            try:
+                return self.context["view"].randomized_qs.index(obj)
+            except ValueError:
+                return obj.standard_index
```

Comment:
Why would this occur?

Reply:
In the detail view the list is empty. The index is incorrect there for shuffled hanging lists. I checked with @chrisvanrun if he uses the index in the detail view, but I'm fairly sure the answer is no.

Reply:
OK, please add a comment and a test. Would returning `None` here to note that it is invalid make sense?

Reply:
A lot more sense probably :slightly_smiling_face: